solana-programs/start.sh

```diff
@@ -177,13 +177,14 @@
 # Back up 2 directories to audius-protocol/solana-programs
 cd ../../
 
+# TODO follow up PR on cleaning up anchor admin storage naming
 cat <<EOF
 {
     "anchorProgramId": "$anchor_program_id",
-    "anchorAdminPublicKey": "$admin_keypair_publickey",
-    "anchorAdminPrivateKey": "$admin_keypair_privatekey",
-    "anchorAdminStoragePublicKey": "$admin_storage_keypair_publickey",
-    "anchorAdminStoragePrivateKey": "$admin_storage_keypair_privatekey",
+    "anchorAdminPublicKey": "$admin_authority_keypair_publickey",
```

Comment:
are there references to these values that we have to propagate? in local setup for DP etc.

Reply:
i think so! i've got a follow up PR that will rename even more but wanted to unblock first.
https://github.com/AudiusProject/audius-protocol/pull/3011/files